templates/compose/mautic5.yaml

```diff
@@ -0,0 +1,111 @@
+  mautic_web:
+    image: 'mautic/mautic:latest'
+    ports:
+      - 8880:80
+    volumes:
+      - './mautic/config:/var/www/html/config:z'
+      - './mautic/logs:/var/www/html/var/logs:z'
+      - './mautic/media/files:/var/www/html/docroot/media/files:z'
+      - './mautic/media/images:/var/www/html/docroot/media/images:z'
+      - './cron:/opt/mautic/cron:z'
+    environment:
+      - SERVICE_FQDN_MAUTIC_80
+      - 'DOCKER_MAUTIC_LOAD_TEST_DATA=${MAUTIC_LOAD_TEST_DATA:-false}'
+      - 'DOCKER_MAUTIC_RUN_MIGRATIONS=${MAUTIC_RUN_MIGRATIONS:-false}'
+      - 'MAUTIC_DB_HOST=${MYSQL_HOST}'
+      - 'MAUTIC_DB_PORT=${MYSQL_PORT}'
+      - 'MAUTIC_DB_DATABASE=${MYSQL_DATABASE}'
+      - 'MAUTIC_DB_USER=${MYSQL_USER}'
+      - 'MAUTIC_DB_PASSWORD=${SERVICE_PASSWORD_64_MYSQL}'
+      - 'MAUTIC_MESSENGER_DSN_EMAIL=${MAUTIC_MESSENGER_DSN_EMAIL}'
+      - 'MAUTIC_MESSENGER_DSN_HIT=${MAUTIC_MESSENGER_DSN_HIT}'
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:80"]
+      interval: 2s
+      timeout: 10s
+      retries: 15
+    depends_on:
+      db:
+        condition: service_healthy
+  mautic_cron:
+    image: 'mautic/mautic:latest'
+    links:
+      - 'db:mysql'
+    volumes:
+      - './mautic/config:/var/www/html/config:z'
+      - './mautic/logs:/var/www/html/var/logs:z'
+      - './mautic/media/files:/var/www/html/docroot/media/files:z'
+      - './mautic/media/images:/var/www/html/docroot/media/images:z'
+      - './cron:/opt/mautic/cron:z'
+    environment:
+      - DOCKER_MAUTIC_ROLE=mautic_cron
+      - 'MAUTIC_DB_HOST=${MYSQL_HOST}'
+      - 'MAUTIC_DB_PORT=${MYSQL_PORT}'
+      - 'MAUTIC_DB_DATABASE=${MYSQL_DATABASE}'
+      - 'MAUTIC_DB_USER=${MYSQL_USER}'
+      - 'MAUTIC_DB_PASSWORD=${SERVICE_PASSWORD_64_MYSQL}'
+      - 'MAUTIC_MESSENGER_DSN_EMAIL=${MAUTIC_MESSENGER_DSN_EMAIL}'
+      - 'MAUTIC_MESSENGER_DSN_HIT=${MAUTIC_MESSENGER_DSN_HIT}'
+    depends_on:
+      mautic_web:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:80"]
+      interval: 2s
+      timeout: 10s
+      retries: 15
+  mautic_worker:
+    image: 'mautic/mautic:latest'
+    links:
+      - 'db:mysql'
+    volumes:
+      - './mautic/config:/var/www/html/config:z'
+      - './mautic/logs:/var/www/html/var/logs:z'
+      - './mautic/media/files:/var/www/html/docroot/media/files:z'
+      - './mautic/media/images:/var/www/html/docroot/media/images:z'
+      - './cron:/opt/mautic/cron:z'
+    environment:
+      - DOCKER_MAUTIC_ROLE=mautic_worker
+      - 'MAUTIC_DB_HOST=${MYSQL_HOST}'
+      - 'MAUTIC_DB_PORT=${MYSQL_PORT}'
+      - 'MAUTIC_DB_DATABASE=${MYSQL_DATABASE}'
+      - 'MAUTIC_DB_USER=${MYSQL_USER}'
+      - 'MAUTIC_DB_PASSWORD=${SERVICE_PASSWORD_64_MYSQL}'
+      - 'MAUTIC_MESSENGER_DSN_EMAIL=${MAUTIC_MESSENGER_DSN_EMAIL}'
+      - 'MAUTIC_MESSENGER_DSN_HIT=${MAUTIC_MESSENGER_DSN_HIT}'
+    depends_on:
+      mautic_web:
+        condition: service_health
```

Comment:
There is a typo here: `service_health` should be `service_healthy`

Reply:
Also, I think there is an issue with the version 5 config.  The web process cannot find the DB.

```2024-09-12T16:21:39.681752542Z mysqladmin: connect to server at '' failed
2024-09-12T16:21:39.683000375Z error: 'Can't connect to server on 'localhost' (111)'
2024-09-12T16:21:39.683031602Z Check that mariadbd is running and that the socket: '/run/mysqld/mysqld.sock' exists!```
Is appearing in the logs

Reply:
@JuanxCursed ^

Reply:
@Skeyelab, thanks for the reporting, and sorry take that long to update the PR.

Fix it on commit 1b8365d